service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/question/ApprovalDateTimeBlockPrePopulator.java

```diff
@@ -35,6 +37,6 @@ public OrderQuestionBlock accept() {
             );
         }
 
-        return Map.of();
+        return Map.of("manageOrdersApprovalDateTime", time.now());
```

Comment:
I think you want to check if manageOrdersApprovalDateTime is already present, so you don'r reset it if you use previous, see ApprovalDateBlockPrePopulator